randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1365,37 +1400,34 @@ Extra - asset_id: 115
   * Layers: default
   * Normal Door to Warehouse Zeela Room/Door to Warehouse Kihunter Room
   > Door to Warehouse Save Room
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs
+      Can Lay Any Bombs
   > Door to Baby Kraid Room
       Trivial
   > Pickup (Missile Expansion)
-      Morph Ball and Power Bombs
+      All of the following:
+          Can Lay Power Bombs
+          Spring Ball or Mid-Air Morph (Intermediate)
 
 > Door to Warehouse Save Room; Heals? False
   * Layers: default
   * Normal Door to Warehouse Save Room/Door to Warehouse Kihunter Room
   > Door to Warehouse Zeela Room
```

Comment:
you can also get with only morph to the save room

Reply:
Maybe its patched in VARIA, but in vanilla there is a bomb block.
![image](https://github.com/randovania/randovania/assets/46821362/ee3760a5-d659-40d1-be3c-0385fcf1b58c)

Reply:
from the pickup, not frfom the left
![grafik](https://github.com/randovania/randovania/assets/38186597/a250a13f-58a2-4fe5-a1cc-ab780b6f6d37)

Reply:
theres some rare chance this can become relevant due to how whacky ammo is i believe. better be safe than sorry with this one.

Reply:
Added.